manager/api/transport_test.go

```diff
@@ -0,0 +1,535 @@
+package api_test
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"net/http"
+	"net/http/httptest"
+	"strings"
+	"testing"
+
+	"github.com/mainflux/mainflux/manager"
+	"github.com/mainflux/mainflux/manager/api"
+	"github.com/mainflux/mainflux/manager/mocks"
+	"github.com/stretchr/testify/assert"
+)
+
+const contentType = "application/json; charset=utf-8"
+
+var (
+	validUser       = manager.User{"user@example.com", "password"}
```

Comment:
Please drop "valid" prefix from all valid entities. This will improve readability together with other changes.